frontend/src/components/GlobalAmount/Props.ts

```diff
@@ -0,0 +1,12 @@
+import { Wallet, Card, BankAccount } from './../../models/accounts';
+
+
+export type GlobalAmountPresentationalProps = {
+    total: number;
+}
+
+export type GlobalAmountProps = {
+    bank_accounts: Array<BankAccount>;
```

Comment:
Codacy found an issue: [Identifier 'bank_accounts' is not in camel case.](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5891730)